projects/canopy/src/lib/progress-indicator/progress-bar/progress-bar.component.scss

```diff
@@ -0,0 +1,11 @@
+.lg-progress-bar {
+  background-color: #e3f2fd;
+  height: calc(var(--space-xxxs) / 2);
+  margin-bottom: calc(var(--space-xxxs) / 2);
```

Comment:
Thanks @jaynguyenslng, I see what you mean, but I think that a hardcoded 2px margin on the bottom is not a great idea, as it might cause hard-to-find bugs, such as with vertical alignment inside a container. We have things like the `lg-margin` directive which can take care of adding margins in this scenario. 

I'll drop a message #design-canopy and see if this 2px is meant to be a fixed property that should always be there, or just an artifact of the design files 👍

Reply:
Here is the thread in Slack: https://udclandg.slack.com/archives/CTMDY6N95/p1718299372931449